modules/javafx.graphics/src/main/java/com/sun/javafx/css/parser/CssLexer.java

```diff
@@ -23,57 +23,52 @@
  * questions.
  */
 
-package javafx.css;
+package com.sun.javafx.css.parser;
 
 import java.io.IOException;
 import java.io.Reader;
 import java.util.HashMap;
 import java.util.Map;
 
-import com.sun.javafx.css.parser.LexerState;
-import com.sun.javafx.css.parser.Recognizer;
-import com.sun.javafx.css.parser.Token;
-
-
-final class CssLexer {
-    final static int STRING = 10;
-    final static int IDENT = 11;
-    final static int FUNCTION = 12;
-    final static int NUMBER = 13;
-    final static int CM = 14;
-    final static int EMS = 15;
-    final static int EXS = 16;
-    final static int IN = 17;
-    final static int MM = 18;
-    final static int PC = 19;
-    final static int PT = 20;
-    final static int PX = 21;
-    final static int PERCENTAGE = 22;
-    final static int DEG = 23;
-    final static int GRAD = 24;
-    final static int RAD = 25;
-    final static int TURN = 26;
-    final static int GREATER = 27;
-    final static int LBRACE = 28;
-    final static int RBRACE = 29;
-    final static int SEMI = 30;
-    final static int COLON = 31;
-    final static int SOLIDUS = 32;
-    final static int STAR = 33;
-    final static int LPAREN = 34;
-    final static int RPAREN = 35;
-    final static int COMMA = 36;
-    final static int HASH = 37;
-    final static int DOT = 38;
-    final static int IMPORTANT_SYM = 39;
-    final static int WS = 40;
-    final static int NL = 41;
-    final static int FONT_FACE = 42;
-    final static int URL = 43;
-    final static int IMPORT = 44;
-    final static int SECONDS = 45;
-    final static int MS = 46;
-    final static int AT_KEYWORD = 47;
+public final class CssLexer {
+    public final static int STRING = 10;
```

Comment:
minor: `public static final` maybe?

Reply:
I want to keep the diffs minimal here, because I didn't change anything except making the constants public.

Reply:
Changed to canonical modifier order.